ci-operator/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-release-4.5-presubmits.yaml

```diff
@@ -103,7 +103,6 @@ presubmits:
     labels:
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-ovn-kubernetes-release-4.5-e2e-gcp-ovn
-    optional: true
```

Comment:
This too.

Reply:
Checked this too.

https://github.com/openshift/release/blob/5d8e151961b0aa3b1c6cf245c581ab0597392d42/ci-operator/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master-presubmits.yaml#L158-L250